txstratum/toi_client.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass
```

Comment:
I like this client, and it seems to be completely independent from the tx-mining-service. It looks like a good candidate to be its own lib that other services could use.

Reply:
I'm considering this, but we should have 2 clients an async and a sync one (using aiohttp and requests). For now I think we should focus on making the toi-service operational and later we can remove this local client for a lib.